src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java

```diff
@@ -84,4 +88,13 @@ public ResponseEntity<org.springframework.core.io.Resource> preview(Integer medi
         return new ResponseEntity<>(new InputStreamResource(audioMediaFile), HttpStatus.OK);
     }
 
+
+    @Override
+    public ResponseEntity<Void> addAudioMetaData(
+        @Parameter(name = "AddAudioMetaDataRequest", description = "") @Valid @RequestBody(required = false) AddAudioMetaDataRequest addAudioMetaDataRequest) {
```

Comment:
Just spotted this - suspect we haven't got the OAS right in a few other places also for POST.
I think it should be `@RequestBody(required = true)` 
Looks like we need to add a required field under requestBody alongside content level.
See [Request Body Object](https://swagger.io/specification/#request-body-object)

Is there a reason why we need to have the annotations here in the Controller, our team have recently been keeping the generated annotations under interface just `AudioApi` - don't think they need to be copied here? Prefer it be removed and kept in the generated code just.

Reply:
ACCEPT.